dashboard/src/components/ChartView/ChartReadme.scss

```diff
@@ -32,3 +32,7 @@
 .ChartReadme tr {
   border-bottom: 1px solid #eee;
 }
+
+.ChartReadme img {
+  max-width: 100%;
```

Comment:
Guessing you've verified this doesn't affect existing chart views?

Reply:
it does affect charts as well, the thing is that since it's not common to have images in those chart readmes I have never seen an image escaping the component boundaries. With this change it should be fine.